Commands/Commands/queue.js

```diff
@@ -370,6 +370,9 @@ commands.registerVote = {
                     cause: 'chat_vote',
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e)
+                } else if (e.statusCode === 422) {
```

Comment:
because it's nothing with your pr, you followed the style of the old error catching, it was just a comment